dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 -r requirements.txt
-errbot
+errbot~=5.2
 pytest
 pytest-mock
+pluggy==0.11
```

Comment:
Put comments here explaining why those pinnings (for pluggy also link to the relevant issue in pluggy's repo)